Library/Formula/shml.rb

```diff
@@ -0,0 +1,16 @@
+class Shml < Formula
+  desc "Style Framework for The Terminal"
+  homepage "http://shml.xyz"
+  url "https://github.com/MaxCDN/shml/archive/1.0.4.tar.gz"
+  sha256 "583fefbbf2adf45d989b052696be0f20f695903878558a4d2e9f74255b91ed51"
+
+  def install
+    bin.install "shml.sh"
```

Comment:
Some people like to put scripts that need sourcing into the `prefix`, others prefer `libexec`. I have no strong preference either way. If it needs to live in the `bin` as well you'll want to create a symlink there.

The syntax would be:
```ruby
libexec.install "shml.sh"
bin.install_symlink libexec/"shml.sh" => "shml"
```
And then replace `libexec` with `prefix` in that example, if you so wish to.